### DISCUSSION TITLE
Italics tags showing as * or <em> in some spots

### DISCUSSION THREAD

**Discussion by zsofiaj** [thegetty/quire#925] [Get Help]
I believe at least some part of this is a known issue, but I just wanted to document here that the italics tags are showing in our project in some spots, and not setting the text in italics. I've attached three examples of where these show up (see screenshots below): in all search results, in the page title at the top of the browser window, and in the pdf footer. Perhaps these are all related? I've asked about the pdf footer before, and I believe @Erin-Cecele or @geealbers said they were working on that. Please let me know if I should move any part of this into a new post in the Issues section. 

I'm on Quire version 1.0.0-rc.11.

Thank you!

<img width="1434" alt="Screenshot 2024-04-25 at 1 54 17 PM" src="https://github.com/thegetty/quire/assets/68071544/f4ed04d9-68f4-49a7-9898-ad573651fe6a">
<img width="1429" alt="Screenshot 2024-04-25 at 1 54 32 PM" src="https://github.com/thegetty/quire/assets/68071544/89ecbc4b-6c2f-4816-bae4-621d0d6686c9">
<img width="1293" alt="Screenshot 2024-04-25 at 1 56 12 PM" src="https://github.com/thegetty/quire/assets/68071544/8003c72e-367c-4ecc-9190-39c86bb8a1e9">

**Comment by Erin-Cecele** [comment]
1) Regarding `<em>` tags, unfortunately, this issue requires a bit of a heavy lift that involves tinkering with the templates and adding complicated CSS. We would recommend having a developer provide assistance. That said, this is something we are aware of and hope to improve in the future, but we do not have it on the immediate roadmap.

2) Regarding asterisks appearing in place of italics, these are two separate bugs. Please open new issues on the [issues tracker](https://github.com/thegetty/quire/issues: one for the appearance of italics in the search and one for the appearance of italics in the URL. I will explain why this is happening in the issue to keep the discussion centralized. Thank you for catching this!

**Reply by zsofiaj** [reply]
Thank you! I opened two new issues for the issue with the asterisks in the search results and in the page title above the url.

**Reply by Erin-Cecele** [reply]
Thank you @zsofiaj!! You're a power user!! We greatly appreciate you reporting bugs as you come across them to help us improve the Quire experience for all!
